modules/zookeeper/src/test/java/org/apache/ignite/spi/discovery/tcp/ipfinder/zk/curator/TestingZooKeeperServer.java

```diff
@@ -30,11 +31,15 @@
 import java.io.Closeable;
 import java.io.IOException;
 import java.util.Collection;
+import java.util.Properties;
 import java.util.concurrent.atomic.AtomicReference;
 
 /**
  */
 public class TestingZooKeeperServer extends QuorumPeerMain implements Closeable {
+    /** */
+    public static final String SECURITY_CLIENT_PORT = "secureClientPort";
```

Comment:
Lets add link to zookeeper. Like 'zkSecureClientPort'

Reply:
There is no such link in zookeeper project. String is hardcoded.